apps/src/netsim/NetSimRemoteNodeSelectionPanel.js

```diff
@@ -169,7 +169,12 @@ NetSimRemoteNodeSelectionPanel.prototype.render = function () {
   NetSimUtils.makeContinueButton(this);
 
   this.addRouterButton_ = this.getBody().find('#netsim-lobby-add-router');
-  this.addRouterButton_.click(this.addRouterCallback_);
+  this.addRouterButton_.click(function () {
+    if (!this.addRouterButton_.is('[disabled]')) {
+      this.addRouterButton_.attr('disabled', true);
```

Comment:
Right now the lobby view (including the *Add Router* button) fully re-renders when we detect that lobby contents have changed, which puts the button back in an enabled state.  This is clumsy - I guess I should store this button's disabled state on the controller instead of the DOM element, and correctly re-enable it when the request returns?